tests/Tutor.Web.Tests/Integration/TutorTestFactory.cs

```diff
@@ -50,6 +58,16 @@ private static void InitializeDbForTests(TutorContext db)
             db.Database.ExecuteSqlRaw(startingDb);
         }
 
+        private static void InitializeEventDbForTests(EventContext db)
+        {
+            var createScript = db.Database.GenerateCreateScript();
+            try
+            {
+                db.Database.ExecuteSqlRaw(createScript);
+            }
+            catch (Exception) { }
+        }
```

Comment:
Known issue with initialization of the event database for tests: this method will attempt to execute the create script without checking if the table already exists. The tests still execute correctly and the event database is initialized, but the unnecessary failed create calls slightly slow down initialization. 
The implementation was left like this for now since we can't use EnsureCreated to initialize the EventContext because both contexts use the same database, and EF Core doesn't seem to have a method for checking if a specific table actually exists.